.env.development

```diff
@@ -1,8 +1,8 @@
 # These are the default development settings.
 # To make changes to your specific instance, please modify .env
 
-CANONICAL_HOST=localhost
-CANONICAL_PORT=3000
+# CANONICAL_HOST=localhost
+# CANONICAL_PORT=3000
```

Comment:
What was the reason for removing these? (I don't know either why they're there, or why we'd take them out :D)